factorial_02.py

```diff
@@ -0,0 +1,10 @@
+num1=input("Enter number: ")
+try:
+    def factorial(num): #Using Recursive Method
+        if num == 0 or num ==1:
+            return 1
+        else:
+            return num * factorial(num-1)
```

Comment:
Please add comment about return keyword at the end of program

Reply:
#The return keyword is to exit a function and return a value.